src/public/modules/users/controllers/authentication.client.controller.js

```diff
@@ -239,25 +244,32 @@ function AuthenticationController(
         }, 500)
 
         GTag.loginSuccess('otp')
-      },
-      function (error) {
+      })
+      .catch((error) => {
+        let errorMsg = get(
+          error,
+          'response.data',
+          'Failed to send login OTP. Please try again later and if the problem persists, contact us.',
+        )
+        const errorStatus = get(error, 'response.status')
+        if (errorStatus >= StatusCodes.INTERNAL_SERVER_ERROR) {
+          errorMsg = 'An unknown error occurred'
+        }
```

Comment:
would a ternary operator be better here or would it be too long to be readable? 

```ts
const errorMsg = (errorStatus >= StatusCodes.INTERNAL_SERVER_ERROR) ? 'An unknown error occurred' : get(
          error,
          'response.data',
          'Failed to send login OTP. Please try again later and if the problem persists, contact us.',
        )
```

Reply:
too many brackets and commas, poor readability, prefer the current implementation.